synfig-studio/src/gui/app.cpp

```diff
@@ -1346,11 +1347,16 @@ App::get_default_accel_map()
 
 	return default_accel_map;
 }
+Glib::RefPtr<App> App::create() {
+	app_reference = Glib::RefPtr<App>(new App("", nullptr, nullptr));
+	return app_reference;
+}
 
 /* === M E T H O D S ======================================================= */
+App::App(const synfig::String& basepath, int *argc, char ***argv) :
```

Comment:
You can now remove the constructor arguments as they are no longer used.

Reply:
Understood !